app/services/vendor_status.rb

```diff
@@ -0,0 +1,78 @@
+class VendorStatus
+  def initialize(from: nil, sp: nil)
+    @from = from
+    @sp = sp
+  end
+
+  IAL2_VENDORS = %i[acuant lexisnexis_instant_verify lexisnexis_trueid].freeze
+  ALL_VENDORS = (IAL2_VENDORS + %i[sms voice]).freeze
+
+  def vendor_outage?(vendor)
+    status = case vendor
+    when :acuant
+      IdentityConfig.store.vendor_status_acuant
+    when :lexisnexis_instant_verify
+      IdentityConfig.store.vendor_status_lexisnexis_instant_verify
+    when :lexisnexis_trueid
+      IdentityConfig.store.vendor_status_lexisnexis_trueid
+    when :sms
+      IdentityConfig.store.vendor_status_sms
+    when :voice
+      IdentityConfig.store.vendor_status_voice
```

Comment:
These methods don't exist yet. I imagine they'd raise? Should we leave em out here, or add the configs?

Reply:
Added in a7f78bc since it was convenient to use them in the specs.